src/xpcc/architecture/driver/atomic/queue.hpp

```diff
@@ -67,6 +67,9 @@ namespace xpcc
 			ALWAYS_INLINE bool
 			isFull() const;
 			
+			ALWAYS_INLINE bool
+			isNotFull() const { return not isFull(); }
```

Comment:
I think niklas introduced them and since he is the main developer of the code they start to prevail. I myself like the more common ! more, since it is shorter and i am used to reading code with ! && and ||